tests/integration/imported-targets/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.10)
+
+project(import-targets
+    VERSION 0.1
+    DESCRIPTION "Tests greentea-client's exported targets can be imported"
+    LANGUAGES C CXX
+)
+
+find_package(greentea-client)
```

Comment:
Is there any expectation (like an established convention) that the package name matches the namespace name?

Reply:
Usually it is the case that package name matches the namespace name. `fmt` and `Boost` both seem to use this convention at least. Should we change the package name to "greentea"?

Reply:
Sounds good to me